core/src/main/java/tech/pegasys/web3signer/core/config/BaseConfig.java

```diff
@@ -64,4 +64,6 @@ public interface BaseConfig {
   boolean keystoreParallelProcessingEnabled();
 
   int getVertxWorkerPoolSize();
+
+  boolean isGenericSigningExtensionEnabled();
```

Comment:
This is going to apply to both eth1 and eth2 modes, we should move this up into eth2 config